src/main/java/com/checkmarx/flow/custom/ADOIssueTracker.java

```diff
@@ -235,7 +235,7 @@ else if(!ScanUtils.empty(request.getApplication())){
         CreateWorkItemAttr title = new CreateWorkItemAttr();
         title.setOp("add");
         title.setPath(Constants.ADO_FIELD.concat(TITLE_FIELD));
-        title.setValue(getXIssueKey(resultIssue, request));
+        title.setValue(request.getProduct().getProduct() + " " + getXIssueKey(resultIssue, request));
```

Comment:
```
request.getProduct().getProduct() + " " + getXIssueKey(resultIssue, request)
```
This snippet now appears 5 times throughout the code. I suggest that we put this logic into a single place (HTMLHelper?)

Reply:
Done.